extensions/amp-timeago/0.1/amp-timeago.js

```diff
@@ -68,15 +68,16 @@ export class AmpTimeAgo extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
-    observeWithSharedInOb(this.element, (inViewport) =>
-      this.viewportCallback_(inViewport)
+    this.unobserveIntersections_ = observeIntersections(
+      this.element,
+      ({isIntersecting}) => this.viewportCallback_(isIntersecting)
     );
     return Promise.resolve();
   }
 
   /** @override */
   unlayoutCallback() {
-    unobserveWithSharedInOb(this.element);
+    this.unobserveIntersections_?.();
```

Comment:
Null this out after cleanup to allow GC:
```suggestion
    this.unobserveIntersections_?.();
    this.unobserveIntersections_ = null;
```

Reply:
Done.